web/src/main/java/org/akaza/openclinica/control/MainMenuServlet.java

```diff
@@ -168,10 +168,11 @@ public void processRequest() throws Exception {
                 }
 
                 ////Integer assignedDiscrepancies = getDiscrepancyNoteDAO().countAllItemDataByStudyAndUser(currentStudy, ub);
-                //Integer assignedDiscrepancies = getDiscrepancyNoteDAO().getViewNotesCountWithFilter(" AND dn.assigned_user_id ="
-                //  + ub.getId() + " AND (dn.resolution_status_id=1 OR dn.resolution_status_id=2 OR dn.resolution_status_id=3)", currentStudy);
```

Comment:
do you know why this code was originally commented out?

Reply:
no, but I check the code when change study will call the same logic, so this same logic has been used and tested somewhere else.

Reply:
looks like this may have been done to improve performance of the MainMenu page. Reverting this change might affect the performance. Can you check what the performance impact of this is by testing using a customer's database backup?

Reply:
tested on one customer's database backup, login as site user, can't feel the impact on performance,  feel no difference, login less than 1 sec

Reply:
ok thanks